gssapi/tests/test_high_level.py

```diff
@@ -381,8 +417,19 @@ def test_sasl_properties(self):
             if mech.description:
                 self.assertIsInstance(mech.description, str)
 
-            cmp_mech = gssmechs.Mechanism.from_sasl_name(mech.sasl_name)
-            self.assertEqual(str(cmp_mech), str(mech))
+            # Heimdal fails with Unknown mech-code on sanon
+            if not (
```

Comment:
Please don't write if statements this way.  Project style is closer to:

```python
            if not (self.realm.provder.lower() == "heimdal" and
                    s = "1.3.6.1.4.1.5322.26.1.110"):
```

Reply:
Sorry force of habit is to use this style (black has trained me too well :)). I'll fix it up.